src/app/static/src/tests/components/downloadResults/uploadModal.test.ts

```diff
@@ -163,10 +164,11 @@ describe(`uploadModal `, () => {
         return store;
     };
 
-    const getWrapper = () => {
+    const getWrapper = (directives = {}) => {
```

Comment:
I'd expect to see tests in this file that the uploadFilesToADR action is dispatched with expected createRelease flag values.